website/blog/2025-04-11-run-ferretdb-postgres-documentdb-extension-cnpg-kubernetes.md

```diff
@@ -80,7 +80,7 @@ metadata:
   namespace: cnpg
 spec:
   instances: 3
-  imageName: 'ghcr.io/ferretdb/postgres-documentdb:17-0.102.0-ferretdb-2.1.0'
+  imageName: 'ghcr.io/ferretdb/postgres-documentdb:17-0.104.0-ferretdb-2.3.1'
```

Comment:
Thanks for contributing to this - please update it to the latest version of DocumentDB and FerretDB (`ghcr.io/ferretdb/postgres-documentdb:17-0.106.0-ferretdb-2.5.0` and `ghcr.io/ferretdb/ferretdb:2.5.0`)

Reply:
Thank you, done!